tutorials/polkadot-sdk/build-a-parachain/connect-a-parachain.md

```diff
@@ -0,0 +1,331 @@
+
+4. Fill in the required fields
+    1. **id** - type the parachain identifier you reserved
+    2. **genesisHead** - click the **file upload** button and select the `para-2000-genesis-state` file you exported
+    3. **validationCode** - click the **file upload** button and select the `para-2000-wasm` file you exported
+    4. **paraKind** - select **Yes** if you are registering a parachain or **No** if you are registering a parathread
+    5. Click on the **Submit Transaction** button
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/connect-a-parachain/connect-a-parachain-8.webp)
+
+5. After submitting the transaction, you can navigate to the **Explorer** tab and check the list of recent events for successful `paras.PvfCheckAccepted`
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/connect-a-parachain/connect-a-parachain-9.webp)
+
+    After the parachain is initialized, you can see it in **Parachains** section of the Polkadot.Js Apps interface
+
+6. Click Network and select Parachains and wait for a new epoch to start
+
+   The relay chain tracks the latest block—the head—of each parachain. When a relay chain block is finalized, the parachain blocks that have completed the validation process are also finalized. This is how Polkadot achieves pooled, shared security for its parachains.
+
+   After the parachain connects to the relay chain in the next epoch and finalizes its first block you can see information about it in the Polkadot/Substrate Portal.
+
+   The terminal where the parachain is running also displays details similar to the following:
+
+## Resetting the Blockchain State
+
+The parachain collator you connected to the relay chain in this tutorial contains all of the blockchain data for the parachain. There's only one node in this parachain network, so any transactions you submit are only stored on this node. Relay chains don't store any parachain state. The relay chain only stores header information for the parachains that connect to it.
+
+For testing purposes, you might want to purge the blockchain state to start over periodically. However, you should remember that if you purge the chain state or manually delete the database, you won’t be able to recover the data or restore the chain state. If you want to preserve data, you should ensure you have a copy before you purge the parachain state.
+
+If you want to start over with a clean environment for testing, you should completely remove the chain state for the local relay chain nodes and the parachain.
+
+To reset the blockchain state, follow these steps:
+
+1. In the terminal where the parachain template node is running, press `Control-c`
+
+2. Purge the parachain collator state by running the following command
+
+    ```bash
+     ./target/release/parachain-template-node purge-chain --chain raw-parachain-chainspec.json
+    ```
+
+3. In the terminal where either the alice validator node or the bob validator node is running, press `Control-c`
```

Comment:
You must remove the data from both every node, they wont be able to sync if they have different data in the dbs

Reply:
What do you mean here? I get that having outdated or un-sync data in each db would not let this work for further node syncs, but I think this is solved in step 4.

Reply:
In the 3rd step, you mention that either Alice OR Bob can purge their state. But if you want to reset the network state and both nodes to be able to sync after the reset, all the nodes have to purge their db